From c1fedab2f807f74ae83bf43b049ca1276dd7a350 Mon Sep 17 00:00:00 2001
From: Mauro Rossi <issor.oruam@gmail.com>
Date: Sun, 16 Aug 2020 11:52:07 +0200
Subject: [PATCH 085/106] android: Kconfig: replace '---help---' with 'help'

Fixes the following building error happenign with kernel 5.9:

drivers/android/Kconfig:52: syntax error
drivers/android/Kconfig:51: unknown statement "---help---"
---
 drivers/android/Kconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/android/Kconfig b/drivers/android/Kconfig
index 87099076d701..2e3f210dda0a 100644
--- a/drivers/android/Kconfig
+++ b/drivers/android/Kconfig
@@ -48,7 +48,7 @@ config ANDROID_BINDER_IPC_32BIT
 	bool "Use old (Android 4.4 and earlier) 32-bit binder API"
 	depends on ANDROID_BINDER_IPC
 	default !64BIT
-	---help---
+	help
 	  The Binder API has been changed to support both 32 and 64bit
 	  applications in a mixed environment.
 
-- 
2.17.1

